lib/recurly/adjustment.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Recurly.Adjustment do
+  @moduledoc """
+  Module for handling adjustments in Recurly.
+  See the [developer docs on adjustments](https://dev.recurly.com/docs/adjustment-object)
+  for more details
+  """
+  use Recurly.Resource
+  alias Recurly.Resource
+
+  @endpoint "accounts/<%= account_code %>/adjustments"
+
+  schema :adjustment do
+    field :account_code,          :string
+    field :accounting_code,       :string
+    field :currency,              :string
+    field :description,           :string
+    field :end_date,              :date_time
+    field :quantity,              :integer
+    field :revenue_schedule_type, :string
+    field :start_date,            :date_time
+    field :tax_code,              :boolean
+    field :tax_exempt,            :string
+    field :unit_amount_in_cents,  :integer
+  end
+
+  @doc """
+  Creates an adjustment from a changeset.
+
+  ## Parameters
+
+  - `changeset` Keyword list changeset
+  - `account_code` String account code of associated account
+
+  ## Examples
+
+  ```
+  alias Recurly.ValidationError
+
+  case Recurly.Adjustment.create(account_code: "myaccountcode", unit_amount_in_cents: 100, currency: "USD") do
+    {:ok, adjustment} ->
+      # created the adjustment
+    {:error, %ValidationError{errors: errors}} ->
+      # will give you a list of validation errors
+  end
+  ```
+  """
+  def create(changeset) do
```

Comment:
I think it should be:

```elixir
def create(changeset, account_code)
```

`changeset` is the first argument because that is the primary data that you are working with. Makes piping more intuitive:

```
{:ok, account} = Recurly.Account.first()

changeset_data = [...]

changeset_data |> some_filter |> Recurly.Adjustment.create(account.account_code)
```